### DISCUSSION TITLE
Welcome to blog Discussions!

### DISCUSSION THREAD

**Discussion by digoal** [digoal/blog#90] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RUDS13031** [comment]
Hi wanted some help with PolarFS configuration in shared storage.

**Reply by digoal** [reply]
you can send issue in this repo: https://github.com/ApsaraDB/PolarDB-FileSystem
